### DISCUSSION TITLE
Help with simplifying the slack message content

### DISCUSSION THREAD

**Discussion by mike-credisense** [datalust/seq-tickets#2156] [Help] [answered]
I see in the slack app integration there is the option to put custom messaging., Though there's not much in the way of documentation/examples to look at to figure out how to change the message outside of the pointer to the slack documentation.

Is anyone able to provide an example message field contents that still maintains the link to seq, the exception details and 1 property from the log entry but excludes all other details? We need this for data sensitivity issues not exposing too much in slack but driving the user back to seq

**Comment by nblumhardt** [comment, marked as answer]
Hi Mike,

Is the "Included properties" field the one you're looking for?

![image](https://github.com/datalust/seq-tickets/assets/342712/42479a60-6aef-4046-91a7-e477ed1c2231)

E.g. you'd put the one property name you want to include, here, overriding the default all-properties behavior.

**Reply by mike-credisense** [reply]
What changes can be made to the message field?

![image](https://github.com/datalust/seq-tickets/assets/89108029/1b5d74cd-821e-4dc6-9fd9-966fba83a570)

**Reply by nblumhardt** [reply]
Hi! I think they're `Level`, `EventType`, `RenderedMessage`, `Exception`, and any custom event properties - e.g. `[Level] [RenderedMessage]`.

Here's where the substitution is performed:

https://github.com/bytenik/Seq.App.Slack/blob/master/src/Seq.App.Slack/Formatting/EventFormatting.cs#L52

The overall message fields are fairly static, they're added by: https://github.com/bytenik/Seq.App.Slack/blob/master/src/Seq.App.Slack/Messages/DefaultMessageBuilder.cs#L11

Hope this helps,
Nick
